tools/free_energy/alchemical_run/gmx_fep.xml

```diff
@@ -0,0 +1,518 @@
+            </when>
+        </conditional>
+        <conditional name="feoptions">
+            <param name="feop" type="select" label="Free energy variables">
+                <option value="default">Use default options</option>
+                <option value="modify">Modify default options</option>
+            </param>
+            <when value="default"/>
+            <when value="modify">
+            <param name="scalpha" type="float" value="0.5" label="The soft-core alpha parameter (sc-alpha)" />
+            <param name="scrpower" type="integer" value="6" label="The power of the radial term in the soft-core equation (sc-r-power)" />
+            <param name="scpower" type="integer" value="1" label="The power for lambda in the soft-core function (sc-power)" />
+            <param name="sccoul" type="text" value="no" label="Apply the soft-core free energy interaction transformation to the Columbic interaction of a molecule (sc-coul)? "/>
+            <param name="scsigma" type="float" value="0.3" label="The soft-core sigma" />
+            <param name="couplelambda0" type="text" value="vdw-q" label="Interactions at lambda=0" help="options are vdw-q, vdw, q, or none"/>
+            <param name="couplelambda1" type="text" value="none" label="Interactions at lambda=1" help="options are vdw-q, vdw, q, or none"/>
+            <param name="coupleintramol" type="text" value="no" label="Turn off intra-molecular interactions?" />
+            </when>
+        </conditional>
+        <param name="temperature" type="float" value="300.0" label="Temperature /K" />
+        <param name="pressure" type="float" value="1.0" label="Pressure /bar" />
+        <param name="input_fep" type="select" label="Generate inputs files only?">
+            <option value="no">No</option>
+            <option value="yes">Yes</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="fepinpout" format="tar" from_work_dir="input_files.tar" label="Input files"/>
+        <data name="dataout" format="tar" from_work_dir="data.tar" label="TI/FEP data output"/>
+        <data name="trajout" format="tar" from_work_dir="traj.tar" label="Trajectories output"/>
+        <data name="report" format="txt" label="Report"/>
+    </outputs>
+    <tests>
+         <test>
+            <param name="groin" value="morph.gro" ftype="gro"/>
+            <param name="topin" value="morph.top" ftype="top"/>
+            <param name="minstep" value="10"/>
+            <param name="nvtstep" value="50"/>
+            <param name="nptstep" value="50"/>
+            <param name="mdstep" value="2000"/>
+            <param name="seed" value="19880924"/>
+            <param name="cons" value="No"/>
```

Comment:
check capitalisation for all test params; I don't know for certain but I assume Galaxy will not recognise `No` and `no` as the same

Reply:
```suggestion
            <param name="cons" value="no"/>
```

Reply:
like this

Reply:
ping @tsenapathi. Actually it would be ideal to run tests with different values to the defaults set. Then you can be certain the tests are not passing by accident like here.